cmd/runner/main.go

```diff
@@ -54,7 +57,12 @@ func parseArgs() {
 	flag.DurationVar(&interval, "interval", 0, "Minimum duration between two non concurrent requests")
 	// global timeout
 	flag.DurationVar(&globalTimeout, config.FieldGlobalTimeout, 0, "Max duration of test")
-
+	// request method
+	flag.StringVar(&method, config.FieldMethod, "", "HTTP request method")
+	// body type
+	flag.StringVar(&bodyType, config.FieldBodyType, "", "HTTP body type")
+	// body content
+	flag.StringVar(&bodyContent, config.FieldBodyContent, "", "HTTP request body content as JSON (you must escape double quotes in the object)")
```

Comment:
I think a single flag for the two values would be more practical to use, something like
```sh
benchttp -body 'raw:{"id":12}'
benchttp -body file:./my/image.jpg
```
(file is just for the example)
That way we have to parse the value internally, but it avoids having fields required or not based on other fields.
It's not a problem to me that it differs from the config file structure in this case.

Reply:
Yep that the idea!

> `	// out of scope?`

Yes let's not bother with files and consider it an eternal _future improvement_ 😛